### PR TITLE
MILKER changes and additions

### PR DESCRIPTION
**Pull request by Jackofashepard**
## About The Pull Request

As it says on the tin. Bigger milkers bigger milk. 
Pregnancy also increases amount
Renamed to Milk tits instead of force milk tits
Milking is no longer tied to climax and climax limit
Milk comes out in smaller amounts depending on when last milked in 5 minute intervals
Added breast milk so people cant make $$$$$$$ off of milk tankers
Seelies produce way less for obvious reasons

## Why It's Good For The Game

I feel MILKER size needs accurate in game representation in form of increased MILK output

### CODE REVIEW

**Comment by wow50000** [comment]
butcher has fallen, millions must SA

**Comment by Jackofashepard** [comment]
> Add a timer to milking that reduces how much it produces, along with knocking down the total milk production in general

There's already a timer in that its wired to CUM code so you can only milk 3 times a day and it takes a bit to milk someone since their climax stat or whatever needs to build up

**Comment by wow50000** [comment]
image
this feels like an accident waiting to happen

**Comment by Jackofashepard** [comment]
Alright its stable and tested

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -213,6 +213,30 @@
...
+		. = 1
+	if(holder.has_reagent(/datum/reagent/consumable/capsaicin))
+		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(!HAS_TRAIT(H, TRAIT_NOHUNGER))
+			H.adjust_hydration(10)
+			H.adjust_nutrition(5)
+		if(H.blood_volume < BLOOD_VOLUME_NORMAL)
+			H.blood_volume = min(H.blood_volume+10, BLOOD_VOLUME_NORMAL)
```
Comment (Cyprex) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:237]: Where'd you grab this code from, assuming it's from somewhere else?

File: code/datums/sexcon/sexcon.dm [hunk 2]
```diff
@@ -216,12 +216,65 @@
...
+		if(vagina.pregnant)
+		{
+			milk_amount = milk_amount + 20
+		}
+	}
+	return milk_amount = round((milk_amount * min(((world.time - breasts.last_milked)/(5 MINUTES)), 1)), 1)
+
 /datum/sex_controller/proc/milk_container(obj/item/reagent_containers/glass/C)
+	var/obj/item/organ/breasts/breasts = user.getorganslot(ORGAN_SLOT_BREASTS)
+	var milk_amount
```
Comment (Cyprex) [inline code/datums/sexcon/sexcon.dm:270]:
```suggestion
	var/milk_amount
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 1 above)
Comment (Jackofashepard) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm:237]: Milk, its a copy paste of milk

**Comment by BlueMemesauce** [comment]
So if your hungry you can just milk yourself and drink it? maybe like milking increase your hunger and if your starving you cant milk

File: code/datums/sexcon/sexcon.dm [hunk 3]
```diff
@@ -216,12 +216,65 @@
...
 /datum/sex_controller/proc/milk_container(obj/item/reagent_containers/glass/C)
+	var/obj/item/organ/breasts/breasts = user.getorganslot(ORGAN_SLOT_BREASTS)
+	var/milk_amount
 	log_combat(user, user, "Was milked into a container")
 	user.visible_message(span_lovebold("[user] lactates into [C]!"))
 	playsound(user, 'sound/misc/mat/endout.ogg', 50, TRUE, ignore_walls = FALSE)
-	C.reagents.add_reagent(/datum/reagent/consumable/milk, 15)
-	after_ejaculation()
+	milk_amount = calculate_milk()
+	C.reagents.add_reagent(/datum/reagent/consumable/breastmilk, milk_amount)
```
Comment (Squird5) [inline code/datums/sexcon/sexcon.dm:275]:
```suggestion
	C.reagents.add_reagent(/datum/reagent/consumable/breastmilk, milk_amount)
	user.adjust_hydration(-(10))
	user.adjust_nutrition(-(5))
```

**Review by Squird5 — CHANGES_REQUESTED** [review]

File: code/datums/sexcon/sexcon.dm (hunk 3 above)
Comment (Squird5) [reply code/datums/sexcon/sexcon.dm:275]: I'd also suggest giving basic milk the 5 nutrition to keep them even

**Pull request closed without merging**